### PR TITLE
Major redesign: Update page titles. Delete tacky typewriting animation. Introduce minimalist theme to site via css. Create transition for homepage title.

### PR DESCRIPTION
**Pull request by gannerr**

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -16,10 +16,11 @@ body {
   font-family: Poppins;
   font-weight: 400;
   color: white;
+  background: black;
 }
 
 html { 
-  background: url("/images/portfolioMainPageGIF.gif") no-repeat center center fixed; 
+  /*background: url("/images/portfolioMainPageGIF.gif") no-repeat center center fixed; */
```
Comment (terencedignon) [inline portfolio/src/main/webapp/style.css:23]: Why is this commented out? It's probably best to remove if it isn't being used.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -36,6 +37,53 @@ h1 {
 
 * {
   box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: sans-serif;
+}
+
+.titlecontainer{
```
Comment (terencedignon) [inline portfolio/src/main/webapp/style.css:45]: nit: I've always seen CSS with a space between the declaration and the '{'.

**Comment by gannerr** [comment]
Screenshot 2020-06-19 at 2 45 29 PM
Screenshot 2020-06-18 at 5 40 36 PM

before and after

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (gannerr) [reply portfolio/src/main/webapp/style.css]: got it, fixed

File: portfolio/src/main/webapp/style.css (hunk 1 above)
Comment (gannerr) [reply portfolio/src/main/webapp/style.css]: makes sense, fixed

**Comment by terencedignon** [comment]
Looks great! Thank you for the screenshots.

**Pull request merged**